Automattic/wp-calypso
# Domains: A/B test only one primary button on domain suggestions

#### Changes proposed in this Pull Request

* This test gives only the best matched domain a primary button; the best alternative gets a secondary button, and the rest of the suggestions get borderless buttons. This helps give the information a visual hierarchy.

**allPrimary (control):**

<img width="809" alt="screen shot 2019-01-10 at 2 39 10 pm" src="https://user-images.githubusercontent.com/2124984/50992605-a3390480-14e5-11e9-8651-03967e8e7d43.png">

**onePrimary:**

<img width="979" alt="image" src="https://user-images.githubusercontent.com/448298/51363677-1f63b700-1aa7-11e9-8289-58c991a97f57.png">

#### Testing instructions

* Switch to this PR and navigate to `/domains/add` or `/start/domains`
* Assign yourself to the `allPrimary` variation of the `domainSearchButtonStyles` test
* Check that all the buttons on the domain suggestions are primary
* Switch to the `onePrimary` variation
* Check that only the best matched domain is primary and the rest are secondary buttons
